IDEOorg/steps
# #161240607 Resolve failing tests in TaskList.test.tsx

#### What does this PR do?
- This PR resolves failing tests in TaskList.test.tsx
#### Description of Task to be completed?
- Install react-router-enzyme-context package
- Modify tests in TaskList.test.tsx
- Update snapshot TaskList.test.tsx.snap
#### How should this be manually tested?
- Run `npm test`
- Observe the tests in `apps/admin/src/routes/Client/TaskList.test.tsx` passing
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#161240607](https://www.pivotaltracker.com/story/show/161240710)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
